commit b96d9b3b09f0427b289332c6f6bfbf747a19b654
Author: Chao Yu <chao@kernel.org>
Date:   Fri Aug 6 10:45:20 2021 +0800

    f2fs: fix to keep compatibility of fault injection interface
    
    The value of FAULT_* macros and its description in f2fs.rst became
    inconsistent, fix this to keep compatibility of fault injection
    interface.
    
    Fixes: 67883ade7a98 ("f2fs: remove FAULT_ALLOC_BIO")
    Signed-off-by: Chao Yu <chao@kernel.org>
    Signed-off-by: Jaegeuk Kim <jaegeuk@kernel.org>

diff --git a/fs/f2fs/f2fs.h b/fs/f2fs/f2fs.h
index 13a7cfe9b23f..4b6ea498a1e0 100644
--- a/fs/f2fs/f2fs.h
+++ b/fs/f2fs/f2fs.h
@@ -43,6 +43,7 @@ enum {
 	FAULT_KVMALLOC,
 	FAULT_PAGE_ALLOC,
 	FAULT_PAGE_GET,
+	FAULT_ALLOC_BIO,	/* it's obsolete due to bio_alloc() will never fail */
 	FAULT_ALLOC_NID,
 	FAULT_ORPHAN,
 	FAULT_BLOCK,